src/firefly/js/templates/lightcurve/LcResult.jsx

```diff
@@ -143,12 +155,13 @@ const StandardView = ({visToolbar, title, searchDesc, imagePlot, xyPlot, tables,
                         FileGroupProcessor: 'LightCurveFileGroupsProcessor'
                     }}>
                     <ValidationField
+                        style={style}
                         initialState={{
-                            value: 'A sample download',
-                            label: 'Title for this download:'
+                            value: `${m}_Files:${currentTime}`,
```

Comment:
In order to make the label entry shorter and useful, i would suggest to format the current time to a more minimalist version: MM/DD/YY - HH:mm:ss (Local time). No need of the day label, and no need of the time zone mention either. 
Example today: 02/20/18 17:36:27
JavaScript toLocaleString() can help for that.

Reply:
I will make the time stamp shorter